src/serialize.jl

```diff
@@ -0,0 +1,28 @@
+const _pickle = PyNULL()
+
+function pickle()
+    if _pickle.o == C_NULL
+        if pyexists(:cPickle)
+            copy!(_pickle, pyimport(:cPickle))
+        else
+            copy!(_pickle, pyimport(:pickle))
+        end
```

Comment:
Isn't cPickle always available in Python 2, and never available in Python 3?  So you could just do:
```jl
pickle() = _pickle.o == C_NULL ? copy!(_pickle, PyCall.pyversion.major ≥ 3 ? "pickle" : "cPickle") : _pickle
```